brunton_lab_to_nwb/brunton_lab_to_nwb.py

```diff
@@ -197,8 +198,11 @@ def run_conversion(
 
 
 def convert_dir(in_dir, n_jobs=1):
-    in_files = glob(os.path.join(in_dir, '*.h5'))
-    out_files = [os.path.splitext(x)[0] + '.nwb' for x in in_files]
+    all_files = Path(in_dir).iterdir()
+    all_data_files = [x.stem for x in all_files if ".h5" in x.suffix]
+    nwb_files = [x.stem for x in all_files if ".nwb" in x.suffix]
+    in_files = [str(in_dir / f"{x}.h5") for x in all_data_files if x not in nwb_files]
+    out_files = [str(in_dir / f"{Path(x).stem}.nwb") for x in in_files]
 
     Parallel(n_jobs=n_jobs)(
```

Comment:
Thanks for parallelizing

Reply:
Could we also have an `n_jobs` default arg on this function?

Reply:
I'm not the one that added the parallelizing, lol, was that way when I cloned it. `n_jobs` currently defaults to `1`. (But I use `12` on smaug)